test/csn_bridge.sh

```diff
@@ -10,7 +10,8 @@ set -Eeuo pipefail
 
 GENPROOFS="$1"
 IBDSIM="$2"
-TEST_DATA=$(mktemp -d)
+TEST_DATA="$3"
```

Comment:
Could still assign a temp dir if $3 is not given.

Reply:
Ok would be helpful for testing locally. I'll add this

Reply:
Added